### DISCUSSION TITLE
can we replace captcha with 2 captcha extention?

### DISCUSSION THREAD

**Discussion by lord-dubious** [david96182/ninjemail#11] [Ideas]
can we simply swap out the captcha extention

**Comment by david96182** [comment]
The project is planned to include support for additional captcha services. However, in order to add support for 2captcha, it is necessary to conduct testing and evaluate its success rate. If 2captcha proves capable of solving the current challenges faced by Yahoo and Outlook, it could be integrated into the project. Please note that utilizing 2captcha may require handling certain aspects of the challenges.

**Comment by meyafex** [comment]
2captcha can't solve funcaptcha(always in outlooks/hotmails registration)
